src/main/java/de/dennisguse/opentracks/util/StringUtils.java

```diff
@@ -130,6 +130,16 @@ public static String formatCoordinate(double coordinate) {
         return Location.convert(coordinate, Location.FORMAT_DEGREES) + COORDINATE_DEGREE;
     }
 
+    /**
+     * Formats a complete coordinate (latitude, longitude)
+     *
+     * @param latitude
+     * @param longitude
+     */
+    public static String formatCoordinate(double latitude, double longitude) {
+        return Location.convert(latitude, Location.FORMAT_DEGREES) + COORDINATE_DEGREE + ", " + Location.convert(longitude, Location.FORMAT_DEGREES) + COORDINATE_DEGREE;
```

Comment:
We may need a string.xml entry for this as the order might be different per lanuage.

Reply:
(incl. the degree sign)